glean-core/rlb/src/dispatcher/mod.rs

```diff
@@ -107,6 +107,9 @@ struct DispatchGuard {
     /// Whether to queue on the preinit buffer or on the unbounded queue
     queue_preinit: Arc<AtomicBool>,
 
+    /// The number of items that were added to the queue after it filled up.
+    overflow_count: Arc<AtomicUsize>,
```

Comment:
Can you kindly add a changelog entry?

Reply:
I did ... then I rebased ... and lost it -_-

Reply:
You almost had a perfect PR. Almost.